docs/contracts-intro.md

```diff
@@ -2,11 +2,159 @@
 
 ---8<-- 'includes/data-builder-danger-header.md'
 
-This page will introduce OpenSAFELY Contracts, explaining:
+## OpenSAFELY Contracts are a clinical data specification
 
-* what they are
-* the process by which new contracts could be proposed
-* where [details of existing contracts will be
-  found](contracts-reference.md)
+*OpenSAFELY Contracts* are an important OpenSAFELY concept that is
+introduced with Data Builder.
```

Comment:
I find this formulation slightly awkward. Partly because of the slippage from plural to singular, but mainly because of "introduced with".

Maybe something like:

> An OpenSAFELY "Contract" is a core concept in the OpenSAFELY framework. Anyone who uses Data Builder should spend time familiarising themselves with how they work.

?

I'm not normally picky but I think the opening sentence is super important!